spec/requests/applications/applications_request_spec.rb

```diff
@@ -37,7 +37,7 @@
 end
 
 feature 'Show application' do
-  let :app do
+  given :app do
```

Comment:
I see. Let's keep them as request specs, rather than features, please. That way we can upgrade and changeset will be smaller, if we need/want to change the type of spec we can do it in follow-up PRs.
Thank you!